cuqi/experimental/mcmc/_hmc.py

```diff
@@ -260,9 +258,8 @@ def step(self):
                 self.current_point = point_prime
                 self.current_target_logd = logd_prime
                 self.current_target_grad = np.copy(grad_prime)
-                self._acc.append(1)
-            else:
-                self._acc.append(0)
+                acc = acc or 1
```

Comment:
Can you explain this change? Seems acc is initially 0, so will always be set to 1 here. Why the or?

Reply:
Thanks @nabriis for spotting this typo. Indeed, this line should be `acc = 1`.